### ISSUE TITLE
No new size information is shown if Read Video Info step is present in the flow

### ISSUE THREAD

**Issue by achurak** [revenz/FileFlows#22]
I'm trying to use the Renamer node, but it appears it can only leverage video related variables (resolution and codec for example) if the Read Video Info node goes right before it, however, in this case the old/new size statistics is missing for the file after the flow is completed (the new video information replaces the original one?).

**Comment by revenz** [comment]
Can you show a screenshot of the flow, or post your flow?

"Read Video Info" was only recently added to refresh the video info after a file has been processed/transcoded.  For most users they probably don't need this.   

File size etc is updated whenever the working file changes.

**Comment by achurak** [comment]
![image](https://user-images.githubusercontent.com/12438813/160528929-8f1b4e37-f296-4748-976c-271604467b89.png)

**Comment by achurak** [comment]
Also, here's my Renamer's content: 
```{file.Name}-{vi.Resolution}_{vi.Video.Codec}{ext}```
Like I said, the "vi." variables are not available if the Read File Info is not there.

**Comment by revenz** [comment]
Yes that's correct usage.   I'm going to update Video Encoder nodes to always refresh that info, so you won't need that Read Video Info soon.  but for now, after processing you do need it otherwise it has the original video info content. 

Are you just saying 
![image](https://user-images.githubusercontent.com/958400/160529543-d0572900-260f-4f11-9d6e-f68fbde8a95c.png)
These Sizes are blank?   If you use that specific flow?

**Comment by achurak** [comment]
> Are you just saying ![image](https://user-images.githubusercontent.com/958400/160529543-d0572900-260f-4f11-9d6e-f68fbde8a95c.png) These Sizes are blank? If you use that specific flow?

Yep, they are blank whenever that "read info" node is present in the flow.

**Comment by achurak** [comment]
![image](https://user-images.githubusercontent.com/12438813/160530028-5084a532-e649-49d4-a0c2-3dd1a6656d51.png)

**Comment by revenz** [comment]
thanks.  I'll take a look at it when I get a chance

**Comment by revenz** [comment]
I cannot reproduce this.   Can you export your flow and post it?

I've tried the following but afterwards it has the correct size and all info as expected.
![image](https://user-images.githubusercontent.com/958400/160555410-3e3b43d0-2646-4185-9f9f-fa5579b700ff.png)

**Comment by achurak** [comment]
Not so sure it's related to that specific node anymore as it seems to work fine on some files as you can see here (4 more files processed and two of them do have size info):
![image](https://user-images.githubusercontent.com/12438813/160643325-05d0e8b8-fb7b-4a54-b5f5-e46e25fcfc33.png)

Here's the exported flow: https://pastebin.com/u21qZhAR

Please note that I had to modify it slightly compared to the screenshot above. Apparently the scaler node would process a file even if it's already the correct resolution and the correct codec (I would expect it to either skip it or, better yet, have two outputs, one for when the codec/resolution match and one for when they don't). I had to add a codec/resolution check in the beginning of the flow to deal with that.

**Comment by achurak** [comment]
So it keeps showing/not showing size seemingly randomly, wondering if it's a problem with the web interface rather than the backend. Here's the log excerpt from one of such files:
<pre>
2022-03-30 19:17:40.3699 - INFO -> File moved successfully
2022-03-30 19:17:40.3754 - INFO -> Initing new moved file
2022-03-30 19:17:41.1324 - INFO -> Connection_Closed
2022-03-30 19:17:42.2987 - INFO -> Final Fingerprint: 67f52b8ef0b9ee70744a026d7427e787c286a78fba0b2d600e970696b605c409
2022-03-30 19:17:42.3009 - INFO -> <b>Original Size: 79069140</b>
2022-03-30 19:17:42.3025 - INFO -> <b>Final Size: 11009780</b>
2022-03-30 19:17:42.3091 - INFO -> Output Path: /media/tv.shows/kids.shows/Peppa Pig/Season 2/Peppa Pig S02E37-720p_hevc.mkv
2022-03-30 19:17:42.3305 - INFO -> Making request[POST]: http://fileflows.media.svc.k3s.churak.com:5000/api/worker/work/finish
2022-03-30 19:18:00.3079 - INFO -> Exit Code: 0
==============================================================================
===                       PROCESSING NODE OUTPUT END                       ===
==============================================================================
</pre>
It does show both the original and the final sizes, but then the file Info (summary) only shows the original size for some reason:
![image](https://user-images.githubusercontent.com/12438813/160920991-39b5791d-4a71-4c27-af1b-5068277a8eb7.png)

Is there anything else I can do on my side to help you troubleshoot?

**Comment by revenz** [comment]
does it always show 0 B for final size on that file, or is it random?

**Comment by revenz** [comment]
Try version 0.4.3.669.  I added a check to not update if the final size reported is 0.  And I added this extra logging line
```
2022-04-02 21:38:58.9296 - INFO -> Recording final size for '10056014005' for 'D:\videos\unprocessed\sample.mkv'
```
This appears in the main system log (Help \ Log) in FileFlows and not in the library file log.

Let me know your results

**Comment by revenz** [comment]
closing as no update.

**Comment by achurak** [comment]
Sorry, I was out of town this whole week. I've just tested with the latest available version (0.4.4.671) and it's still the same. As for the new logging, I see the new line for the files that have their size shown and I don't see that line for the files missing their size.
